src/js/index.js

```diff
@@ -1,15 +1,29 @@
-//import react into the bundle
 import React from "react";
 import ReactDOM from "react-dom";
 
-//include bootstrap npm library into the bundle
-import "bootstrap";
+import "../styles/Index.scss";
 
-//include your index.scss file into the bundle
-import "../styles/index.scss";
+import Counter from "./component/Counter.jsx";
 
-//import your own components
-import Home from "./component/home.jsx";
+let counter = 0;
 
-//render your react application
-ReactDOM.render(<Home />, document.querySelector("#app"));
+let renderCounter = () => {
+	// Calculates a module to obtain every digit
+	const fourth = Math.floor(counter / 1000) % 10;
+	const third = Math.floor(counter / 100) % 10;
+	const second = Math.floor(counter / 10) % 10;
+	const first = counter % 10;
+	counter++;
+	ReactDOM.render(
+		<Counter
+			firstDigit={first}
+			secondDigit={second}
+			thirdDigit={third}
+			fourthDigit={fourth}
+		/>,
+		document.querySelector("#app")
+	);
+};
+
+// SetInterval to render the component every one second
```

Comment:
Please remove this comment, is really obvious that you have just configured and interval.

Reply:
I have removed the unnecesary comment and the unnecesary tag "React.Fragment" from the code.